src/panels/Scene/GlobeLayers/LayersList.tsx

```diff
@@ -0,0 +1,66 @@
+import { useState } from 'react';
+import { DragDropContext, Draggable, Droppable, DropResult } from '@hello-pangea/dnd';
+
+import { useOpenSpaceApi } from '@/api/hooks';
+
+import { GlobeLayer } from './GlobeLayer';
+
+interface Props {
+  layers: string[]; // List of URIs
```

Comment:
What were the opinions you guys had about it?   I've thought about the same thing, and it might be suitable to do it as part of this PR since it relies so heavily on uris.

Reply:
Conclusion: We decided to do this. I made a PR for adding the type aliases to master: https://github.com/OpenSpace/OpenSpace-WebGui/pull/26

Should apply it to this PR once that is merged